### PR TITLE
Collecting basic logs

### PR DESCRIPTION
**Pull request by jovirkku**
Signed-off-by: Jorma Virkkunen <jorma.virkkunen@jolla.com>

### CODE REVIEW

**Comment by jovirkku** [comment]
If our goal here is to move this document from ZD to SailfishOS Docs and check the text of the document then this PR should be ok.
However, if we also want to review the actual script then a different approach is needed as GitHub does not reveal the contents of the zipped script.

**Comment by Thaodan** [comment]
> If our goal here is to move this document from ZD to SailfishOS Docs and check the text of the document then this PR should be ok. However, if we also want to review the actual script then a different approach is needed as GitHub does not reveal the contents of the zipped script.

The script could go to a separate repo, for each version tagged there will be a tarball to download that would make the process more transparent and contributions to the script get easier.

File: Support/Help_Articles/Collecting_Logs/Collect_Basic_Logs/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+NOTE: The data is collected from the previous device restart onwards. If you need to get and study persistent data (data from beyond the restart) then consider [this document](/Support/Help_Articles/Collecting_Logs/Collect_Persistent_Logs).
+
+
+# Preparations
+
+1) Enable the **[Developer mode](/Support/Help_Articles/Enabling_Developer_Mode/)** on your phone. You need to do this only to get the Terminal app and to set up the password for "SSH and root access".
+
+2) Download the script file below (right-click it and take "save link as")
+
+**[collect-logs.sh](/Support/Help_Articles/Collecting_Logs/Collecting_Basic_Logs/collect-logs.sh)**
```
Comment (vigejolla) [inline Support/Help_Articles/Collecting_Logs/Collect_Basic_Logs/README.md:24]: Path is wrong here, it should be "Collect_Basic_Logs". On the other hand, you could probably leave the entire path out, and just have `[collect-logs.sh](collect-logs.sh)` here. Didn't try though.

File: Support/Help_Articles/Collecting_Logs/Collect_Basic_Logs/collect-logs.sh [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+tar -cf $THERESULT $TMPDIR/*.*  2> /dev/null
+chown 100000:100000 $THERESULT   ## handles both nemo and defaultuser
+echo "" 
+ITISHERE=$(pwd)
+echo "---------------------------------------------"
+echo "Get the file '$THERESULT'"
+echo "from $ITISHERE."
+echo "---------------------------------------------"
+echo ""
+
```
Comment (vigejolla) [inline Support/Help_Articles/Collecting_Logs/Collect_Basic_Logs/collect-logs.sh:149]: Quite a lot of empty rows here, perhaps they could be cleaned out?

File: Support/Help_Articles/Collecting_Logs/Collect_Basic_Logs/README.md (hunk 1 above)
Comment (jovirkku) [reply Support/Help_Articles/Collecting_Logs/Collect_Basic_Logs/README.md:24]: A good catch, thanks

**Review by vigejolla — CHANGES_REQUESTED** [review]
Please fix the link

File: Support/Help_Articles/Collecting_Logs/Collect_Basic_Logs/collect-logs.sh (hunk 2 above)
Comment (jovirkku) [reply Support/Help_Articles/Collecting_Logs/Collect_Basic_Logs/collect-logs.sh:149]: They are there intentionally, to give rhythm to the output == make it easier to read. Like I commented, I removed many of them already  :-)

Comment (vigejolla) [reply Support/Help_Articles/Collecting_Logs/Collect_Basic_Logs/collect-logs.sh:149]: Tried it, works well, collected lot's of valuable information.

**Review by vigejolla — APPROVED** [review]

**Pull request merged**